src/status_im2/common/nickname_drawer/view.cljs

```diff
@@ -0,0 +1,84 @@
+(ns status-im2.common.nickname-drawer.view
+  (:require [clojure.string :as string]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icons]
+            [quo2.core :as quo]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.common.nickname-drawer.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn empty?-nickname?
```

Comment:
I believe we don't need to specify `?` at every word in the function name. It can be added only at the end: `empty-nickname?`

Reply:
This is a typo .. Apologies for that